new-integration-tests/pom.xml

```diff
@@ -92,6 +92,11 @@
             <artifactId>assertj-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+          <groupId>commons-io</groupId>
+          <artifactId>commons-io</artifactId>
+          <version>2.6</version>
```

Comment:
Check the pattern... we're using dependency management.  The dependency goes here without the version number, but with <scope>test</scope>.  Then, there is a matching dependency in the dependency management section below where the version comes from a property.

Reply:
fixed.